src/client/clientobject.h

```diff
@@ -84,6 +88,9 @@ class ClientActiveObject : public ActiveObject
 	virtual bool directReportPunch(v3f dir, const ItemStack *punchitem = nullptr,
 		float time_from_last_punch = 1000000) { return false; }
 
+	void selectIfInRange(const Shootline &shootline,
+			std::vector<DistanceSortedActiveObject> &dest);
```

Comment:
I doubt it belongs here.

Reply:
It's pretty short; I can probably inline it.

Reply:
I've looked at it again, and it's not that short. Where do you think it belongs?